dev-db/mariadb/mariadb-11.7.2.recipe

```diff
@@ -0,0 +1,160 @@
+SUMMARY="Maria DB is an open-source relational database management system (RDBMS)"
+DESCRIPTION="MariaDB server is a community developed fork of MySQL server. \
+Started by core members of the original MySQL team, MariaDB actively works \
+with outside developers to deliver the most featureful, stable, and sanely \
+licensed open SQL server in the industry."
+HOMEPAGE="https://www.mariadb.org/"
+COPYRIGHT="2009 - 2025 MariaDB Foundation"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/MariaDB/server/archive/refs/tags/mariadb-11.7.2.tar.gz"
+CHECKSUM_SHA256="63abb60682d7c62f0195e8aa85301775af11ef0b7e700775dd8f232068fbe448"
+SOURCE_DIR="server-mariadb-$portVersion"
+SOURCE_URI_2="https://github.com/mariadb-corporation/mariadb-connector-c/archive/refs/tags/v3.4.4.tar.gz"
+CHECKSUM_SHA256_2="79a1c5a3de86e7daa0a456c502d60dc15debe105932ad6a0d25024908603f433"
+SOURCE_URI_3="https://github.com/codership/wsrep-lib/archive/70cd967f5e249b53d6cce90e2e4198641c564381.tar.gz"
+CHECKSUM_SHA256_3="bd44ee719f8fb303e8f90aaab105db836edc44cd8bb73004f20810248a0e6758"
+SOURCE_URI_4="https://github.com/codership/wsrep-API/archive/refs/heads/master.tar.gz"
+CHECKSUM_SHA256_4="ed0866ee46043b0e6f23cab0a7bcf11441add76ab9091e9e10c658ff0789841c"
+SOURCE_URI_5="https://github.com/mariadb-corporation/libmarias3/archive/0d5babbe46f17147ed51efd1f05a0001017a2aad.tar.gz"
+CHECKSUM_SHA256_5="6a190a80c210fc68b6590703ccbd5c68199c366786242c4b51b3a011cf7c10cd"
```

Comment:
The last 2 provide stable releases which I think should be used instead of a commit checkout.